openedx/adg/lms/registration_extension/forms.py

```diff
@@ -158,3 +161,24 @@ class ExtendedProfileForm(ModelForm):
     class Meta(object):
         model = ExtendedUserProfile
         fields = ('company',)
+
+
+def get_registration_form_override(request, form_desc):
```

Comment:
Lets please rename this method, This is not the exact override but it just adds defaults. Maybe `get_registration_form_defaults | initials`.

Reply:
No, from name it seems like its the replacement of `get_registration_form`, We are not creating any kind of form but we are just adding some defaults.

Reply:
Oh you mean it seems its overriding method not form values

Reply:
Yes :)